chapters/part3/fairness.md

```diff
@@ -0,0 +1,733 @@
+This section only provides a high-level overview of fairness and how NLP models can <em>encode</em> and <em>amplify</em> biases. We highly encourage readers to consult the works of leading scholars (particularly Black women) to dive further into this field. Some excellent papers <em>in the NLP domain</em> with which to begin your journey include:
+
+1. Blodgett, S.L., Barocas, S., Daumé, H., & Wallach, H. (2020). Language (Technology) is Power: [A Critical Survey of "Bias" in NLP](https://api.semanticscholar.org/CorpusID:218971825). ACL.
+
+2. Chang, K.W., Ordonez, V., Mitchell, M., Prabhakaran, V (2019). [Tutorial: Bias and Fairness in Natural Language Processing](http://web.cs.ucla.edu/~kwchang/talks/emnlp19-fairnlp/). EMNLP 2019.
+
+3. Barocas, S., Hardt, M., and Narayanan, A. 2019. [Fairness and machine learning](https://fairmlbook.org).
+
+4. Bender, E.M., Gebru, T., McMillan-Major, A., & Shmitchell, S. (2021). [On the Dangers of Stochastic Parrots: Can Language Models Be Too Big? 🦜](https://api.semanticscholar.org/CorpusID:232040593). Proceedings of the 2021 ACM Conference on Fairness, Accountability, and Transparency.
+
+In the rest of this section, we motivate the need for fairness and bias mitigation tools in NLP, based on References 1, 2 and 3 above.
+
+<em>Fairness</em> concerns ensuring that a model performs equitably for all groups of people with respect to protected attributes. For instance, we may want a model to correctly predict `[MASK]` tokens at the same rate for men, women, and non-binary genders.
+
+<em>Biases</em> are stereotypical/unjust associations a model encodes with respect to protected attributes, or subpar model performance for certain groups. Hence, <em>bias mitigation</em> is the process of lessening the severity of stereotypical/unjust associations and disparate model performance. For example, we may want to disassociate all occupations from gender, to mitigate gender bias in a downstream application that recommends jobs to individuals. 
+
+## Biases in Data
+
+We are all aware that large language models are hungry for text data, and these data greatly influence the word and sentence-level representations the models learn. NLP researchers and pratictioners often scrape data from the Internet, which include news articles, Wikipedia pages, and even patents! But, how representative are the text in these data of the real-world frequences of different words or groups of people? The answer is **not at all**.
+<br>
+<br>
+This is the result of <em>reporting bias</em>, in which the "frequency with which people write about actions, outcomes, or properties is not a reflection of real-world frequencies or the degree to which a property is characteristic of a class of individuals" [2]. For instance, "murdered" occurs with a much higher frequency than does "blinked" in text corpuses because journalists usually report about murders rather than blinking, ironically as a result of the relative rarity of murders [2].
```

Comment:
I would drop the word `ironically` as it's not particularly unexpected or surprising.